WordPress/src/main/java/org/wordpress/android/ui/stats/StatsSinglePostDetailsActivity.java

```diff
@@ -0,0 +1,711 @@
+
+            // show the trophy indicator if the value is the maximum reached
+            if (currentDay.getCount() == maxReachedValue) {
+                holder.imgMore.setVisibility(View.VISIBLE);
+                holder.imgMore.setImageDrawable(getResources().getDrawable(R.drawable.ic_action_like_active));
+                holder.totalsTextView.setTextColor(getResources().getColor(R.color.calypso_orange));
+                holder.imgMore.setBackgroundColor(Color.TRANSPARENT); // Hide the default click indicator
+            } else {
+                holder.imgMore.setVisibility(View.GONE);
+                holder.totalsTextView.setTextColor(getResources().getColor(R.color.stats_text_color));
+            }
+
+            holder.networkImageView.setVisibility(View.GONE);
+            return convertView;
+        }
+
+        @Override
+        public int getChildrenCount(int groupPosition) {
+            PostViewsModel.Week week = groups.get(groupPosition);
+            return week.getDays().size();
+        }
+
+        @Override
+        public Object getGroup(int groupPosition) {
+            return groups.get(groupPosition);
+        }
+
+        @Override
+        public int getGroupCount() {
+            return groups.size();
+        }
+
+
+        @Override
+        public long getGroupId(int groupPosition) {
+            return 0;
+        }
+
+        @Override
+        public View getGroupView(final int groupPosition, boolean isExpanded,
+                                 View convertView, ViewGroup parent) {
+
+            if (convertView == null) {
+                convertView = inflater.inflate(R.layout.stats_list_cell, parent, false);
+                convertView.setTag(new StatsViewHolder(convertView));
+            }
+
+            final StatsViewHolder holder = (StatsViewHolder) convertView.getTag();
+
+            PostViewsModel.Week week = (PostViewsModel.Week) getGroup(groupPosition);
+
+            int total = week.getTotal();
+
+            // change the color of the text if one of its childs has reached maximum value
+            int numberOfChilds = getChildrenCount(groupPosition);
+            boolean shouldShowTheTrophyIcon = false;
+            for (int i=0; i < numberOfChilds; i++) {
+                PostViewsModel.Day currentChild = (PostViewsModel.Day) getChild(groupPosition, i);
+                if (currentChild.getCount() == maxReachedValue) {
+                    shouldShowTheTrophyIcon = true;
+                }
+            }
+
+            // Build the label to show on the group
+            String name;
+            PostViewsModel.Day firstChild = (PostViewsModel.Day) getChild(groupPosition, 0);
+            if (numberOfChilds > 1) {
+                PostViewsModel.Day lastChild = (PostViewsModel.Day) getChild(groupPosition, getChildrenCount(groupPosition) - 1);
+                name = StatsUtils.parseDate(firstChild.getDay(), "yyyy-MM-dd", "MMM dd")
```

Comment:
I see the same date formatting strings hard-coded throughout the stats code (not just in this PR). I think it would be helpful to create string constants for these.